csm/core/agent/csm_agent.py

```diff
@@ -35,27 +35,31 @@ class CsmAgent:
     @staticmethod
     def init():
         Conf.load(const.CSM_GLOBAL_INDEX, f"yaml://{const.CSM_CONF}")
+        syslog_port = Conf.get(const.CSM_GLOBAL_INDEX, "Log>syslog_port")
+        backup_count = Conf.get(const.CSM_GLOBAL_INDEX, "Log>total_files")
+        file_size_in_mb = Conf.get(const.CSM_GLOBAL_INDEX, "Log>file_size")
         Log.init("csm_agent",
-               syslog_server=Conf.get(const.CSM_GLOBAL_INDEX, "Log.syslog_server"),
-               syslog_port=Conf.get(const.CSM_GLOBAL_INDEX, "Log.syslog_port"),
-               backup_count=Conf.get(const.CSM_GLOBAL_INDEX, "Log.total_files"),
-               file_size_in_mb=Conf.get(const.CSM_GLOBAL_INDEX, "Log.file_size"),
-               log_path=Conf.get(const.CSM_GLOBAL_INDEX, "Log.log_path"),
-               level=Conf.get(const.CSM_GLOBAL_INDEX, "Log.log_level"))
-        if ( Conf.get(const.CSM_GLOBAL_INDEX, "DEPLOYMENT.mode") != const.DEV ):
+               syslog_server=Conf.get(const.CSM_GLOBAL_INDEX, "Log>syslog_server"),
+               syslog_port= int(syslog_port) if syslog_port else syslog_port,
```

Comment:
Tmm, it might be good to explicitly state here and further what actual value you're supposed to get in case of no syslog_port or backup_count or whatever. Is it `None` or `""` or `0`?

Reply:
Sure will change

Reply:
Done with Changes Made them None since if these values are missing log works without rsyslog using python logger.